submissions/adeyemiJuwon/HTMLforms/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```

Comment:
Your html file in `introduction to HTML` is your index page. This view isn't needed.

Reply:
But that is where the product description is